Text/CustomEntitySearch/WordLinker.cs

```diff
@@ -13,42 +13,60 @@ namespace AzureCognitiveSearch.PowerSkills.Text.CustomEntitySearch
 {
     internal class WordLinker
     {
-        public static WordLinker WordLink(string executingPathDirectory, string fileType)
+        public static WordLinker WordLink(string fileName)
         {
-            if (fileType == "json")
+            var local_root = Environment.GetEnvironmentVariable("AzureWebJobsScriptRoot");
+            var azure_root = $"{Environment.GetEnvironmentVariable("HOME")}/site/wwwroot";
+            var actual_root = local_root ?? azure_root;
+
+            if (fileName.EndsWith(".json"))
             {
-                string json = File.ReadAllText($"{executingPathDirectory}\\words.json");
+                string json = File.ReadAllText(Path.Join(actual_root, fileName));
                 return JsonConvert.DeserializeObject<WordLinker>(json);
             }
-            else if (fileType == "csv")
+            else if (fileName.EndsWith(".csv"))
             {
                 return new WordLinker
                 {
-                    Words = File.ReadAllLines($"{executingPathDirectory}\\words.csv").ToList()
+                    Words = File.ReadAllLines(Path.Join(actual_root, fileName))
+                            .SelectMany(line => line.Split(","))
+                            .Where(line => !string.IsNullOrEmpty(line))
+                            .ToList()
```

Comment:
This is an oversimplification of CSV. We should either clarify in the readme that we only accept comma-separated, no quotes, or bring in one of the simple CSV packages that exist.

Reply:
we may need to take that change after workshops are over. Going to won't fix for the scope of this PR. 

---
In reply to: [330772512](https://github.com/Azure-Samples/azure-search-power-skills/pull/14#discussion_r330772512) [](ancestors = 330772512)

Reply:
I'd still add a comment in the readme and/or a bug to acknowledge we're aware of it.